spark/v3.5/spark-extensions/src/main/scala/org/apache/spark/sql/execution/datasources/v2/ExtendedDataSourceV2Strategy.scala

```diff
@@ -123,6 +128,18 @@ case class ExtendedDataSourceV2Strategy(spark: SparkSession) extends Strategy wi
         allowExisting = allowExisting,
         replace = replace) :: Nil
 
+    case d@DescribeRelation(ResolvedV2View(catalog, ident), _, isExtended, _) =>
+      DescribeV2ViewExec(d.output, catalog.loadView(ident), isExtended) :: Nil
+
+    case show@ShowTableProperties(ResolvedV2View(catalog, ident), propertyKey, _) =>
+      ShowV2ViewPropertiesExec(show.output, catalog.loadView(ident), propertyKey) :: Nil
```

Comment:
It's a little odd to use `show.output` but then name `propertyKey` in the pattern. Why not name `output` in the pattern and reference it directly?

Reply:
you're right, this should be the `output` in the pattern for all of these